src/services/kuma-api/KumaApi.ts

```diff
@@ -142,15 +142,15 @@ export default class KumaApi extends Api {
   }
 
   getAllDataplaneOverviews(params?: DataPlaneOverviewParameters): Promise<PaginatedApiListResponse<DataPlaneOverview>> {
-    return this.client.get('/dataplanes+insights', { params })
+    return this.client.get('/dataplanes/_overview', { params })
   }
 
   getAllDataplaneOverviewsFromMesh({ mesh }: { mesh: string }, params?: DataPlaneOverviewParameters): Promise<PaginatedApiListResponse<DataPlaneOverview>> {
-    return this.client.get(`/meshes/${mesh}/dataplanes+insights`, { params })
+    return this.client.get(`/meshes/${mesh}/dataplanes/_overview`, { params })
   }
 
   getDataplaneOverviewFromMesh({ mesh, name }: { mesh: string, name: string }, params?: any): Promise<DataPlaneOverview> {
-    return this.client.get(`/meshes/${mesh}/dataplanes+insights/${name}`, { params })
+    return this.client.get(`/meshes/${mesh}/dataplanes/${name}/_overview`, { params })
```

Comment:
@lahabana Can I ask you to review the changes specifically in this file (i.e. KumaApi.ts)?

I specifically want to double-check that the path change for Zone Ingresses is correct (i.e. the addition of the dash). I did run these APIs locally and I think all of the above are right. Just making sure.